_docs/_latest/concepts/best-practices.md

```diff
@@ -40,6 +40,135 @@ Scanning for external identities helps stop untrusted users from getting access
+Some documents recommend using [company tag]-[group tag] prefixed to the above,
+opinion today is that this means project names fall “out of date/stale” too
+fast as companies go through reorgs. Consider moving this information to
+project
+[labels](https://cloud.google.com/resource-manager/docs/creating-managing-labels#using_console).
+
+**Group Naming Convention**
+
+Groups are used to manage user membership in IAM policies. A consistent naming
+convention is recommended for enterprises. E.g. gcp-[system name]-[role type] =>
+gcp-costanalytics-prodsupport, gcp-costanalytics-analysts
+
+**Assign Roles to Groups**
+
+[Assign project roles to groups of
+users](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#assign_project_roles_to_groups_of_users).
+Rather than assign individual user accounts roles in IAM, setup a Group. User
+can then request to join a group (via their an automated and audited process)
+to gain access to the GCP projects.
+
+**Deeply Nested Sub Groups**
+
+Nested groups can [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
+group memberships however they can be hard to reason about when determining
+access. IAM Explain might help with this in the future. Deeply nested groups can
+also delay the propagation of projects to users; as a best practice, one level
+works best.
+
+**Organization Policy - Restricted Sharing Org Policy**
+
+Restricted Sharing Org Policy provides the ability to prevent GCP resources from
+being shared with identities outside their organization.
+
+**Resource Management Hierarchy (GCP Project Structure)**
+
+Usage of the organization node, folders and projects should only have enough
+complexity to solve specific requirements. A typical best practice is for a
+customer to create a project per “system or application”, per “environment”
+(DEV, TEST, PROD). Folders are only introduced if distinct business users
+require autonomy (to create projects and manage Organization Policies) in their
+own “space” in GCP or to have distinct groupings of DEV, TEST, PROD with
+different policies applied at the folder level for each environment.
+
+**Quota Management Process**
+
+GCP provides default [quotas](https://cloud.google.com/compute/quotas) based on
+an organization reputation on the platform.  Enterprises will need to define a
+workflow to monitor their quota usage and to apply for more quota when they
+determine that they are nearing their allocation.
+
+
+**Remove Default IAM Organization Policies**
+
+By default, a GCP organization is created allowing all users to create and
+manage projects. For most large organizations, this should be removed.
+Check the [default access control](https://cloud.google.com/resource-manager/docs/default-access-control).
+
+**Consumer Account Strategy**
+
+It is not possible to prevent users from signing up for a consumer account with
+their work email once the domain is registered with Google, see b/65413695. To
```

Comment:
In general, how do we want to handle 'transient' best practices? @mirons-google - should we include these and update once the bug is fixed? It would be cool but it's an extra support load...

Reply:
(but yes, an internal buganizer link is not helpful)